description.md

```diff
@@ -113,17 +113,18 @@ https://www.irregex.vc/posts/a-look-into-the-most-starred-repos
 While GitHub stars are not a definitive measure of impact, it is notable that nine of the SPEC Core Projects rank among the top 0.036% of all repositories on GitHub by stars \citep{gitstar_ranking}.
 The maintainers of each of the $\spcore$ has agreed to become involved with the $\spec$ process; however, many more projects small and large participate and even more adopt the various $\spec$s for their own projects.
 
-In addition to these projects, we also develop and maintain of a growing collection of **$\sptools$**---overseen by a $\toolsteering$---that help projects across the ecosystem, including those supporting the $\spec$s.
-<!--
-$\spp$ supports on the order of fifteen tools, ranging from release management to community organization, and two services: a discussion forum and web analytics.
--->
-The growing list of developer tools includes
+In addition to these projects, we also develop and maintain a growing collection of **$\sptools$**—overseen by the $\toolsteering$ committee—that support projects across the ecosystem, including those implementing and maintaining the $\spec$s.
```

Comment:
projects that maintain the specs... could be correct, checking that that was your intent.